tests/ci/integration/run_nginx_integration.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash -exu
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+source tests/ci/common_posix_setup.sh
+
+# Set up environment.
+
+# SYS_ROOT
+#  - SRC_ROOT(aws-lc)
+#    - SCRATCH_FOLDER
+#      - nginx
+#      - AWS_LC_BUILD_FOLDER
+#      - AWS_LC_INSTALL_FOLDER
+#      - NGINX_BUILD_FOLDER
+
+# Assumes script is executed from the root of aws-lc directory
+SCRATCH_FOLDER="${SRC_ROOT}/NGINX_BUILD_ROOT"
+NGINX_SRC_FOLDER="${SCRATCH_FOLDER}/nginx"
+NGINX_TEST_FOLDER="${SCRATCH_FOLDER}/nginx-tests"
+NGINX_BUILD_FOLDER="${SCRATCH_FOLDER}/nginx-aws-lc"
+NGINX_PATCH_BUILD_FOLDER="${SRC_ROOT}/tests/ci/integration/nginx_patch"
+NGINX_PATCH_TEST_FOLDER="${SRC_ROOT}/tests/ci/integration/nginx_tests_patch"
+AWS_LC_BUILD_FOLDER="${SCRATCH_FOLDER}/aws-lc-build"
+AWS_LC_INSTALL_FOLDER="${NGINX_SRC_FOLDER}/aws-lc-install"
+
+
+mkdir -p ${SCRATCH_FOLDER}
+rm -rf ${SCRATCH_FOLDER}/*
+cd ${SCRATCH_FOLDER}
+
+function nginx_build() {
+  ./auto/configure --prefix="${NGINX_BUILD_FOLDER}" \
+    --with-http_ssl_module \
+    --with-http_v2_module \
+    --with-http_v3_module \
+    --with-stream \
+    --with-stream_realip_module \
+    --with-stream_ssl_module \
+    --with-stream_ssl_preread_module \
+    --with-mail \
+    --with-mail_ssl_module \
+    --with-cc-opt="-I${AWS_LC_INSTALL_FOLDER}/include" \
+    --with-ld-opt="-L${AWS_LC_INSTALL_FOLDER}/lib"
+  make -j
+  make -j install
```

Comment:
I assume the install target depends on the nginx executable target so `make install` will build everything then install it. Much like you can just run `ninja run_tests` for aws-lc: ninja sees if all of the `run_tests` dependencies are up to date and then does the right thing.
```suggestion
  make -j install
```

Reply:
Sure, I just threw the install on there and forgot to prune the commands needed. `make install` was needed so that `nginx-tests` can find the right binary to test against.